apps/alpha-test-app/README.md

```diff
@@ -0,0 +1,27 @@
+# Alpha Test App
+
+During the alpha phase of onyx, we use this minimalistic app to try out onyx components in a user-near environment and detect problems that Storybook can not show.
+This app will be removed once onyx leaves the alpha phase. As it is temporary, it will not be mentioned in our documentation app.
+
+**Important**: To test changes in onyx, make sure to run `pnpm build` in `packages/sit-onyx/package.json`. Only then will the changes be reflected in the Alpha Test App.
```

Comment:
Alternatively we could just change the start command to build sit-onyx before starting the app

Reply:
I'll add that but will keep the info in the README, as people are used to livereload they might not remember that they have to restart ^^